lms/djangoapps/instructor/tests/test_spoc_gradebook.py

```diff
@@ -81,6 +83,33 @@ def setUp(self):
 
         self.assertEquals(self.response.status_code, 200)
 
+    def test_list_report_downloads_by_course_key(self):
+        with patch('instructor_task.models.LocalFSReportStore.links_for') as mock_links_for:
+            mock_links_for.return_value = [
+                ('mock_file_name_1', 'https://1.mock.url'),
+                ('mock_file_name_2', 'https://2.mock.url'),
+            ]
+            response = list_report_downloads_by_course_key(self.course.id)
+
+        expected_response = {
+            "downloads": [
+                {
+                    "url": "https://1.mock.url",
+                    "link": "<a href=\"https://1.mock.url\">mock_file_name_1</a>",
+                    "name": "mock_file_name_1"
+                },
+                {
+                    "url": "https://2.mock.url",
+                    "link": "<a href=\"https://2.mock.url\">mock_file_name_2</a>",
+                    "name": "mock_file_name_2"
+                }
+            ]
+        }
+        self.assertJSONEqual(
```

Comment:
Where is `assertJSONEqual` defined? Why not just use `json.loads` and compare the two dictionaries?